indico/modules/events/evaluation/blueprint.py

```diff
@@ -0,0 +1,21 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.web.flask.wrappers import IndicoBlueprint
+
+evaluation_blueprint = _bp = IndicoBlueprint('evaluation', __name__, template_folder='templates')
```

Comment:
Currently only used to have a template folder inside the module. But eventually when evaluations go to Postgres, it'll contain some url rules, too.